### PR TITLE
Alphabetical Sorting and Markdown generator

### PR DESCRIPTION
**Pull request by zarbod**
db.csv सोऱ्टींग साठी स्क्रिप्ट 

input - db.csv,
output - sorteddb.csv

### CODE REVIEW

File: sort.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import csv
+
+with open('db.csv') as csv_file:
```
Comment (sanketgarade) [inline sort.py:3]: the file name is hardcoded. it will be better if the input file can be passed as an argument to the script.

**Comment by sanketgarade** [comment]
- the output csv file is not needed in the PR.
- the database format is updated on the main branch (added 2 more columns). can you please modify the script to accommodate this?

File: sort.py (hunk 1 above)
Comment (zarbod) [reply sort.py]: I made it into a function with the file as the argument. Is that what you wanted?

Comment (sanketgarade) [reply sort.py]: Yes somewhat like this. I will check in detail later and let you know.

**Comment by zarbod** [comment]
Just wrote the script to convert the csv file to markdown

**Pull request closed without merging**